docs/source/blog/derotation.md

```diff
@@ -0,0 +1,117 @@
+:blogpost: true
+:date: July 1, 2025
+:author: Laura Porta
+:location: London, UK
+:category: Blog
+:language: English
+:image: 1
+
+# Scientific Background
+
+Passive or active rotation of the head (yaw) is an ethologically relevant movement for rodents, enabling them to orient their bodies, avoid obstacles, and locate food sources. In neuroscience, controlled head rotation is a key experimental paradigm for studying the vestibular system and its influence on sensory processing. Electrophysiological recordings have demonstrated the impact of such movements on neural activity in the visual cortex, as shown by [Velez et al. (2018)](https://www.cell.com/neuron/fulltext/S0896-6273(18)30144-2). 
+
+To further connect neural activity to cellular morphology, multiphoton calcium imaging is commonly employed, necessitating the ability to rotate an animal's head under a microscope while acquiring high-resolution movies. This experimental setup introduces significant motion artifacts due to the line-scanning nature of multiphoton imaging, which the derotation package is specifically designed to correct.
+
+---
+
+# `derotation`: a Python package for correcting motion artifacts in rotating multiphoton movies
+
+The `derotation` package provides a robust solution for reconstructing multiphoton movies of rotating samples acquired with a line-scanning microscope at high rotation speeds and low frame rates. When imaging a rotating sample, the line-by-line nature of acquisition introduces significant in-frame geometric distortions, which make later registration not feasible with frame-by-frame derotation methods. This package corrects these artifacts to produce clear, stable movies suitable for standard analysis pipelines like Suite2p.
```

Comment:
This is good + the scientific background makes it very clear, the only thing I might not be sure what a 'multiphoton movie' is, maybe 'movies of rotating samples acquired with a line-scanning multiphoton microscope'? Or it could be defined once